### ISSUE TITLE
BotRanger needs a logo

### ISSUE THREAD

**Issue by thunder-coding** [thunder-coding/BotRanger#12]
BotRanger currently uses the default Discord profile picture. 

Accepted logo formats:
- `*.svg`
- `*.ai` (Adobe Illustrator)
- `*.png`
- `*.jpg`
- `*.jpeg`

Preferred image formats:
- `*.svg`
- `*.ai`

**Comment by ngeshlew** [comment]
@thunder-coding, I can help with this. Maybe you can add more context as to what BotRanger does (in layman)? 🙂

**Comment by thunder-coding** [comment]
First of all, Thanks for your interest in my project.

BotRanger is a cool Discord bot with some capabilities for now which will be expanded in the future.

Here are the things you can currently do with it:

- Get information about a GitHub user/repo.
- Solve a mathematical expression
- And much more features will be added in upcoming releases...


So basically the bot is not complete yet and will change a lot in future.

So, the logo should say that this is a bot account and must be unique.

That's all. I am not good at designing and will probably take a lot of time in creating a logo.

Again, Thanks for your interest

**Comment by thunder-coding** [comment]
![Screenshot_20210131-145143~2](https://user-images.githubusercontent.com/68849076/106379906-254e3b80-63d5-11eb-90e9-e172f817aefa.png)

This is proposed logo

And here's the link to the SVG code.
https://gist.github.com/thunder-coding/8927f895fe83678c857f88aa82b652db

If you have a better logo suggestion be sure to share here. 

This issue will be closed as soon as the  PR will be merged
